src/Pages/Archived/Archived.jsx

```diff
@@ -2,9 +2,14 @@ import "../../css/main.css";
 import "./archived.css";
 import { useEffect } from "react";
 import { useAuth, useArchive } from "../../Context";
-import { ArchivedCard, Sidebar } from "../../Components";
+import { ArchivedCard, Sidebar, DisplayCardEmpty } from "../../Components";
+import { useScrollToTop, useDocumentTitle } from "../../Utils";
 
 export const Archived = () => {
+
+  useDocumentTitle();
+  useScrollToTop();
```

Comment:
doubt: What is this used for?

Reply:
These are the custom hooks. UseScrollToTop scrolls the page to the top when you enter a particular page. useDocumentTitle is to add the title.